src/napari_tiled_browser/models/tiled_subscriber.py

```diff
@@ -0,0 +1,46 @@
+from qtpy.QtCore import QObject, QRunnable, Signal
+
+
+class TiledSubscriberSignals(QObject):
+    finished = Signal()
+    results = Signal(object)
+
+
+class TiledSubscriber(QRunnable):
+    def __init__(
+        self,
+        *,
+        client,
+        **kwargs,
+    ):
+        super().__init__()
+        self.signals = TiledSubscriberSignals()
+        self.client = client
+
+    def run(self):
+        catalog_sub = self.client.subscribe()
+        catalog_sub.child_created.add_callback(on_new_child)
+        catalog_sub.start()
+
+
+def on_new_child(update):
+    "A new child node has been created in a container."
+    child = update.child()
+    print(child)
+    sub = child.subscribe()
+    # Is the child also a container?
+    if child.structure_family == "container":
+        # Recursively subscribe to the children of this new container.
+        sub.child_created.add_callback(on_new_child)
+    else:
+        # Subscribe to data updates (i.e. appended table rows or array slices).
+        sub.new_data.add_callback(on_new_data)
+        # Launch the subscription.
+        # Ask the server to replay from the very first update, if we already
+        # missed some.
+    sub.start_in_thread(1)
```

Comment:
I suppose we need handle the case where replaying from message #1 overlaps with data we already have.

Is there a unique identifier in the update payload to know whether we are receiving data that is already in the table / image stack?